setup.py

```diff
@@ -52,6 +52,13 @@ def get_version():
 '''
 
 
+# The command line script name.  Currently set to "markdown_py" so as not to
+# conflict with the perl implimentation (which uses "markdown").  We can't use
+# "markdown.py" as the default config on some systems will cause the script to
+# try to import itself rather than the library which will raise an error.
```

Comment:
I'm not sure the second sentence is accurate for an entry point. We might want to remove that one.

Reply:
Do you mean the third sentence? The second still applies, I think, as the entry point usually gets installed into the same namespace as a normal script (something on $PATH).

Reply:
Sorry, yes, third sentence.

Reply:
Ok, the third sentence removed.